README.md

```diff
@@ -1 +1,3 @@
-# Gravity-Forms
```

Comment:
Add a nicer readme, please. Please refer to other documents, like quick payments